src/Tizen.NUI.Components/Style/ToastStyle.cs

```diff
@@ -26,6 +26,19 @@ namespace Tizen.NUI.Components
     [EditorBrowsable(EditorBrowsableState.Never)]
     public class ToastStyle : ControlStyle
     {
+        /// <summary>The Duration bindable property.</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty DurationProperty = BindableProperty.Create(nameof(Duration), typeof(uint?), typeof(ToastStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            ((ToastStyle)bindable).duration = (uint?)newValue;
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            return ((ToastStyle)bindable).duration;
+        });
+
+        private uint? duration;
+
```

Comment:
Why use `duration` in these lines?
`duration` is private but never used in this class.
and propertychanged and defaultvaluecreator seems to useless

Reply:
`defaultValueCreator` is used in `BindableObject.GetValue()`.
If `defaultValueCreator` is not declared, the BindableProperty will always return `null`.

#### Sample Code (When propertychanged and defaultvaluecreator are not used) :

```cs
 public static readonly BindableProperty DurationProperty = BindableProperty.Create(nameof(Duration), typeof(uint?), typeof(ToastStyle), null);
```

```cs
ToastStyle toastStyle = new ToastStyle();
toastStyle.SetValue(ToastStyle.DurationProperty, (uint)30);
object duration = toastStyle.GetValue(ToastStyle.DurationProperty);  // duration is null
```